dali/sasha/sadbghk.hpp

```diff
@@ -0,0 +1,7 @@
+#ifndef SADBGHKIF_HPP
+#define SADBGHKIF_HPP
```

Comment:
trivial: I know lots of code uses this approach, but we should use this instead:
```
#pragma once
```